entrypoint.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -l
```

Comment:
I think all of the logic in this file is working. I was testing it using Docker locally, but I'm failing at getting the ssh keys to connect when running this inside the GH Action.